core/lib/contracts/src/lib.rs

```diff
@@ -59,6 +59,8 @@ const LOADNEXT_CONTRACT_FILE: &str =
     "etc/contracts-test-data/artifacts-zk/contracts/loadnext/loadnext_contract.sol/LoadnextContract.json";
 const LOADNEXT_SIMPLE_CONTRACT_FILE: &str =
     "etc/contracts-test-data/artifacts-zk/contracts/loadnext/loadnext_contract.sol/Foo.json";
+const MAILBOX_CONTRACT_FILE: &str =
```

Comment:
please name it "test mailbox contract", right now it gives off impression that it is a usual mailbox